README.md

```diff
@@ -41,13 +41,15 @@ querying logs.
 
 ### Documentation
 
-Documentation for the latest release can be found
-[here](https://github.com/grafana/loki/tree/v1.3.0/docs/README.md). Documentation for
-the unreleased master branch can be found inside the
-[`docs/` folder](./docs/README.md).
+* [master](./docs/README.md)
+* [v1.3.0](https://github.com/grafana/loki/tree/v1.3.0/docs/README.md)
+* [v1.2.0](https://github.com/grafana/loki/tree/v1.2.0/docs/README.md)
+* [v1.1.0](https://github.com/grafana/loki/tree/v1.1.0/docs/README.md)
+* [v1.0.0](https://github.com/grafana/loki/tree/v1.0.0/docs/README.md)
 
+Or here are some shortcuts to commonly used sections from the documentation:
```

Comment:
We can just say "commonly used sections from the latest release (v1.3.0) documentation:" rather than having the extra note below.

WDYT?

Reply:
changed it, PTAL